tests/serialized_cap.json

```diff
@@ -1 +1 @@
-{"att":{"http://example.com/public/photos/":{"crud/delete":[]},"mailto:username@example.com":{"msg/receive":[{"max_count":5,"templates":["newsletter","marketing"]}],"msg/send":[{"to":"someone@email.com"},{"to":"joe@email.com"}]}},"prf":["bafysameboaza4mnsng7t3djdbilbrnliv6ikxh45zsph7kpettjfbp4ad2g2uu2znujlf2afphw25d4y35pq"]}
+{"att":{"http://example.com/public/photos/":{"crud/delete":[{}]},"mailto:username@example.com":{"msg/receive":[{"max_count":5,"templates":["newsletter","marketing"]}],"msg/send":[{"to":"someone@email.com"},{"to":"joe@email.com"}]}},"prf":["zM5K3WPEexRZN9x7ikg5LoijNjU2ytFTGJ6eeruGiN3Q9xkfGpMVei9cz5jMZAFUsmxXi9k"]}
```

Comment:
> `"crud/delete":[{}]`

Is the empty map strictly necessary? EIP-5573 give examples where the array is empty.

Reply:
the ucan spec 0.10 was updated to specify that [an empty array is equivalent to "not allowed"](https://github.com/ucan-wg/spec/blob/0.10/README.md#3263-caveat-array), whereas an array with `{}` is equivalent to "no restriction" :(. You're right about the EIP, that could be amended. I don't necessarily disagree though that it could be unnecessary in different contexts/representations/encodings